e2e/utils/index.ts

```diff
@@ -100,7 +103,7 @@ export function runCreateWorkspace(
     command += ` --defaultBase="${base}"`;
   }
 
-  if (packageManager) {
+  if (packageManager && !useDetectedPm) {
```

Comment:
Isn't `useDetectedPm = false` the same as `packageManger = null`? Do we need an extra flag here?

Reply:
`useDetectedPm` is a flag to **NOT** set `--package-manager`. This is purely for testing purpose. `packageManager` seems to be set (in test env) via `SELECTED_PM` env variable, and the package commands are fetched from that selected package manager (and is default to `npm`). For the purpose of testing the detection, I'd like to **NOT** set `--package-manager` when there is `packageManager` defined, but still able to have  `getPackageManagerCommand` run correctly.